--- dnssec-tools-1.9/validator/libsres/Makefile.in.orig	2009-04-17 17:38:24.000000000 +0200
+++ dnssec-tools-1.9/validator/libsres/Makefile.in	2011-05-20 22:21:07.353636179 +0200
@@ -63,7 +63,7 @@ clean:
 	$(RM) -rf $(LT_DIR)
 
 $(LIBRES): $(LOBJ)
-	$(LIBTOOLLD) -o $@ $(LOBJ) 
+	$(LIBTOOLLD) -o $@ $(LOBJ) -lcrypto
 
 install:
 	$(MKPATH) $(DESTDIR)/$(libdir)
--- dnssec-tools-2.1/validator/libval/Makefile.in.orig	2015-06-28 16:43:43.279673824 +0200
+++ dnssec-tools-2.1/validator/libval/Makefile.in	2015-06-28 17:25:10.899569429 +0200
@@ -65,7 +65,7 @@
 	$(RM) -rf $(LT_DIR)
 
 $(LIBVAL): $(LOBJ)
-	$(LIBTOOLLD) -o $@ $(LOBJ) $(LSRES)
+	$(LIBTOOLLD) -o $@ $(LOBJ) $(LSRES) -lssl -lcrypto -lpthread
 
 install:
 	$(MKPATH) $(DESTDIR)$(libdir)
