--- dnssec-tools-1.13/validator/apps/dnssec-check/mainwindow.cpp.orig	2012-06-21 22:10:13.000000000 +0200
+++ dnssec-tools-1.13/validator/apps/dnssec-check/mainwindow.cpp	2012-06-25 21:15:43.352058436 +0200
@@ -372,7 +372,6 @@
     }
 
     accessURL.addQueryItem("locationDescription", locationDescription);
-    accessURL.addQueryItem(".13");
 
     if (!m_manager) {
         m_manager = new QNetworkAccessManager();
--- dnssec-tools-1.13/validator/apps/dnssec-check/qmlapplicationviewer/qmlapplicationviewer.pri.orig	2012-02-04 01:50:50.000000000 +0100
+++ dnssec-tools-1.13/validator/apps/dnssec-check/qmlapplicationviewer/qmlapplicationviewer.pri	2012-06-27 19:03:22.065139970 +0200
@@ -113,7 +113,7 @@
             QMAKE_EXTRA_TARGETS += first copydeploymentfolders
         }
     }
-    installPrefix = /opt/$${TARGET}
+    installPrefix = /usr
     for(deploymentfolder, DEPLOYMENTFOLDERS) {
         item = item$${deploymentfolder}
         itemfiles = $${item}.files
--- dnssec-tools-1.13/validator/apps/dnssec-nodes/dnssec-nodes.pro.orig	2012-01-23 22:43:27.000000000 +0100
+++ dnssec-tools-1.13/validator/apps/dnssec-nodes/dnssec-nodes.pro	2012-06-26 17:44:21.238964788 +0200
@@ -1,3 +1,4 @@
+PREFIX=/usr
 HEADERS += \
         edge.h \
         node.h \
@@ -48,7 +49,7 @@
 DATADIR =$$PREFIX/share
 INCLUDEPATH += ../../include
 isEmpty(ANDROID_PLATFORM) {
-    LIBS        += -lval-threads -lsres -lnsl -lcrypto -lpthread
+    LIBS        += -L../../libval/.libs -L../../libsres/.libs -lval-threads -lsres -lcrypto -lpthread
     maemo5 {
         INCLUDEPATH += /opt/maemo/usr/include/
     }
@@ -62,8 +63,6 @@
 # path to the harmattan libraries
 contains(MEEGO_EDITION,harmattan): {
     LIBS += -L/scratchbox/users/hardaker/targets/HARMATTAN_ARMEL/usr/lib
-} else {
-    LIBS += -L/usr/local/lib
 }
 
 QT += network
--- dnssec-tools-1.13/validator/apps/dnssec-system-tray/dnssec-system-tray.pro.orig	2011-10-25 22:45:08.000000000 +0200
+++ dnssec-tools-1.13/validator/apps/dnssec-system-tray/dnssec-system-tray.pro	2012-06-30 12:17:05.175285379 +0200
@@ -1,3 +1,4 @@
+PREFIX=/usr
 TARGET=dnssec-system-tray
 
 HEADERS       = dnssec-system-tray.h \
--- dnssec-tools-1.13/validator/apps/lookup/src/src.pro.orig	2012-06-21 22:02:39.000000000 +0200
+++ dnssec-tools-1.13/validator/apps/lookup/src/src.pro	2012-06-26 18:43:52.315481592 +0200
@@ -11,7 +11,7 @@
 INCLUDEPATH += /home/hardaker/src/dnssec/dnssec-tools.git/dnssec-tools/validator/include
 
 isEmpty(ANDROID_PLATFORM) {
-    LIBS        += -lval-threads -lsres -lnsl -lcrypto -lpthread
+    LIBS        += -L../../../libval/.libs -L../../../libsres/.libs -lval-threads -lsres -lcrypto -lpthread
 } else {
     LIBS        += -L/opt/android-external-openssl/lib/
     LIBS        += -L/root/necessitas/android-ndk-r5c/platforms/android-4/arch-arm/usr/lib/
@@ -63,7 +63,7 @@
 unix {
     #VARIABLES
     isEmpty(PREFIX) {
-        PREFIX = /usr/local
+        PREFIX = /usr
     }
 
 BINDIR = $$PREFIX/bin
@@ -77,13 +77,13 @@
 
   target.path =$$BINDIR
 
-  desktop.path = $$DATADIR/applications/hildon
+  desktop.path = $$DATADIR/applications
   desktop.files += $${TARGET}.desktop
 
   service.path = $$DATADIR/dbus-1/services
   service.files += $${TARGET}.service
 
-  iconxpm.path = $$DATADIR/pixmap
+  iconxpm.path = $$DATADIR/pixmaps
   iconxpm.files += ../data/maemo/$${TARGET}.xpm
 
   icon26.path = $$DATADIR/icons/hicolor/26x26/apps
