diff -urNp -x '*.orig' DNSSEC-Tools-dnssec-tools-2.2.3.org/validator/libsres/Makefile.in DNSSEC-Tools-dnssec-tools-2.2.3/validator/libsres/Makefile.in
--- DNSSEC-Tools-dnssec-tools-2.2.3.org/validator/libsres/Makefile.in	2018-08-30 00:25:30.000000000 +0200
+++ DNSSEC-Tools-dnssec-tools-2.2.3/validator/libsres/Makefile.in	2021-05-08 10:27:02.392411486 +0200
@@ -63,7 +63,7 @@ clean:
 	$(RM) -rf $(LT_DIR)
 
 $(LIBRES): $(LOBJ)
-	$(LIBTOOLLD) -o $@ $(LOBJ) 
+	$(LIBTOOLLD) -o $@ $(LOBJ) -lcrypto
 
 install:
 	$(MKPATH) $(DESTDIR)$(libdir)
diff -urNp -x '*.orig' DNSSEC-Tools-dnssec-tools-2.2.3.org/validator/libval/Makefile.in DNSSEC-Tools-dnssec-tools-2.2.3/validator/libval/Makefile.in
--- DNSSEC-Tools-dnssec-tools-2.2.3.org/validator/libval/Makefile.in	2018-08-30 00:25:30.000000000 +0200
+++ DNSSEC-Tools-dnssec-tools-2.2.3/validator/libval/Makefile.in	2021-05-08 10:27:02.392411486 +0200
@@ -65,7 +65,7 @@ clean:
 	$(RM) -rf $(LT_DIR)
 
 $(LIBVAL): $(LOBJ)
-	$(LIBTOOLLD) -o $@ $(LOBJ) $(LSRES)
+	$(LIBTOOLLD) -o $@ $(LOBJ) $(LSRES) -lssl -lcrypto -lpthread
 
 install:
 	$(MKPATH) $(DESTDIR)$(libdir)
